Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ghcr.io/osgeo/gdal:ubuntu-full-3.12.0beta0
+
+RUN apt-get update && apt-get -y install pip tippecanoe git python3.12-venv nano && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /root
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+RUN unzip awscliv2.zip
+RUN ./aws/install
```

Comment:
Wondering, whether we should use https://pypi.org/project/s3/ in #181 so that we don't need to install a system dependency...